diff --git a/codetag.install b/codetag.install
index 60ce882..ef93d46 100644
--- a/codetag.install
+++ b/codetag.install
@@ -9,7 +9,7 @@ function codetag_requirements($phase) {
   $requirements = [];
 
   if ($phase == 'install' || $phase == 'runtime') {
-    $codetag_detected = file_exists(DRUPAL_ROOT . '/libraries/codeTag/plugin.js');
+    $codetag_detected = file_exists(DRUPAL_ROOT . '/libraries/codetag/plugin.js');
 
     if ($codetag_detected) {
       $requirements['codetag'] = [
diff --git a/src/Plugin/CKEditorPlugin/CodeTag.php b/src/Plugin/CKEditorPlugin/CodeTag.php
index 8ae947d..9665ff3 100644
--- a/src/Plugin/CKEditorPlugin/CodeTag.php
+++ b/src/Plugin/CKEditorPlugin/CodeTag.php
@@ -18,7 +18,7 @@ class CodeTag extends CKEditorPluginBase {
    * {@inheritdoc}
    */
   public function getFile() {
-    return 'libraries/codeTag/plugin.js';
+    return 'libraries/codetag/plugin.js';
   }
 
   /**
@@ -35,7 +35,7 @@ class CodeTag extends CKEditorPluginBase {
     return array(
       'Code' => array(
         'label' => $this->t('CodeTag'),
-        'image' => 'libraries/codeTag/icons/code.png',
+        'image' => 'libraries/codetag/icons/code.png',
       ),
     );
   }
